Add image map for user stories overview

eufjan opened the pull request:
I always wanted to click the images, now we can. Tested it locally and it worked fine.

eufjan committed:
Add image map for user stories overview

PMenges reviewed (changes requested):
Awesome. @KorayKirli wanted to look into this initially. I'm not sure if we hit the right coordinates (or if they are relative to something or depending on screen resolution?).

When building the branch the "browsing data" href goes approx. to half width of the figure and accessing data over the other half. 

<img width="2244" height="585" alt="{22B60EF0-FF9A-49D6-8C10-D3F57E564F79}" src="https://github.com/user-attachments/assets/f3ebd981-3074-4823-825a-46f2eb67685f" />

eufjan commented:
I thought they were relative to the image (full size). Will investigate. The error you experienced was during a local build?

eufjan committed:
Add rwdImageMaps jQuery plugin for responsive image maps

- Introduced the rwdImageMaps plugin to enable responsive image maps.
- The plugin recalculates area coordinates based on the actual image size during load and window resize events.
- Included licensing information and author details in the plugin code.

eufjan committed:
fixed broken link

PMenges reviewed (approved):
This works but has the bug that it calculates the map on load of the page. If I open the page on my laptop screen (1080p) and move it to my regular one (4kp) the map coordinates are off.

This issue imo is niche enough that we can go ahead and merge.
